### PR TITLE
Pull request for Issue #247: Allow non-computed named RDDs 

### PR DESCRIPTION
**Pull request by koettert**
Hi, 
we have also just discovered this problem and propose to move the force computation and storage level arguments into the API to let the user decide if the RDD should be precomputed or not.

@velvia: We would appreciate, if this pull request could be merge quickly.

Thanks
The KNIME team

### CODE REVIEW

File: job-server-api/src/spark.jobserver/NamedRddSupport.scala [hunk 1]
```diff
@@ -38,13 +38,16 @@ trait NamedRdds {
...
+   * @param forceComputation if true, forces the RDD to be computed by calling count().
+   * @param storageLevel the storage level to persist the RDD with. Default: StorageLevel.MEMORY_ONLY.
    * @param timeout if the RddManager doesn't respond within this timeout, an error will be thrown.
    * @tparam T the generic type of the RDD.
    * @return the RDD with the given name.
    * @throws java.util.concurrent.TimeoutException if the request to the RddManager times out.
    * @throws java.lang.RuntimeException wrapping any error that occurs within the generator function.
    */
-  def getOrElseCreate[T](name: String, rddGen: => RDD[T])
+  def getOrElseCreate[T](name: String, rddGen: => RDD[T], forceComputation: Boolean = true,
```
Comment (velvia) [inline job-server-api/src/spark.jobserver/NamedRddSupport.scala]: (style nit) please have each parameter on its own line

**Comment by velvia** [comment]
@koettert this looks fine to me, thanks, other than the style nit.  Want to push a quick change and comment and i'll merge it?

**Comment by TimMaltGermany** [comment]
@velvia @koettert fixed style issue

**Comment by noorul** [comment]
LGTM @TimMaltGermany 

@velvia Merging it

**Comment by noorul** [comment]
Thanks @TimMaltGermany

**Pull request merged**